README.md

```diff
@@ -1,2 +1,120 @@
-# Redis Model Store
+# 🧠 Redis Model Store
 
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+![Language](https://img.shields.io/github/languages/top/redis-applied-ai/redis-model-store)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+![GitHub last commit](https://img.shields.io/github/last-commit/redis-applied-ai/redis-model-store)
+[![pypi](https://badge.fury.io/py/redisvl.svg)](https://pypi.org/project/redis-model-store/)
+
+Store, version, and manage your ML models in Redis with ease. `redis-model-store` provides a simple yet powerful interface for handling machine learning model artifacts in Redis.
+
+## ✨ Features
+
+- **🔄 Automatic Versioning**: Track and manage multiple versions of your models
+- **📦 Smart Storage**: Large models are automatically sharded for optimal storage
+- **🔌 Pluggable Serialization**: Works with any Python object (NumPy, PyTorch, TensorFlow, etc.)
+- **🏃‍♂️ High Performance**: Efficient storage and retrieval using Redis pipelining
+- **🛡️ Safe Operations**: Atomic operations with automatic cleanup on failures
+
+## 🚀 Quick Start
+
+### Installation
+
+```bash
+# Using pip
+pip install redis-model-store
+
+# Or using poetry
+poetry add redis-model-store
+```
+
+### Basic Usage
+
+Here's a simple example using scikit-learn:
+
+```python
+from redis import Redis
+from model_store import ModelStore
```

Comment:
Opinion: I don't like when the pip install and the import differ because it creates confusion. If I pip install `redis-model-store` I expect to `import redis_model_store` or in this case `from redis_model_store import ModelStore`. 

From this import I can immediately tell:
1. where it came from
2. that it's not a local module

When running the workshop the other day, I had several people get stuck because they were trying to pip install `dotenv` instead of `python-dotenv` as another example.

Reply:
Yeah that's fair, and would be consistent with Redis+Python packages for the most part:

`redisvl` and `redis` both follow this. I did want to be a bit clear about the title instead of using an acronym and leaving the user even more confused.

Reply:
Fixed